src/app/icd/BUILD.gn

```
@@ -54,29 +55,33 @@ source_set("sender") {
   ]
 
   public_deps = [
-    ":cluster",
     ":notifier",
+    ":table",
     "${chip_root}/src/credentials:credentials",
     "${chip_root}/src/lib/address_resolve:address_resolve",
     "${chip_root}/src/protocols/secure_channel",
   ]
 }
 
-# ICD management cluster source-set is broken out of the main source-set to enable unit tests
-# All sources and configurations used by the ICD management cluster need to go in this source-set
-source_set("cluster") {
+# ICDMonitoringTable source-set is broken out of the main source-set to enable unit tests
+# All sources and configurations used by the ICDMonitoringTable need to go in this source-set
+source_set("table") {
```

table seems unclear.
```suggestion
source_set("MonitoringTable") {
```

---

done